src/coreclr/interop/platform.h

```diff
@@ -22,6 +22,15 @@
 
 #endif // _WIN32
 
+#ifdef HOST_UNIX
+#include <objidl.h> // COM interfaces
+
+// Common macro for working in COM
+#define RETURN_IF_FAILED(exp) { hr = exp; if (FAILED(hr)) { _ASSERTE(false && #exp); return hr; } }
```

Comment:
Can we have a single definition for these macros?

Reply:
Ok, I'm going to merge them in a single if